README.md

```diff
@@ -29,7 +29,7 @@ The instrumentations in this repo are:
 - strictly complies with [open telemetry semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions)
 - up to date with latest SDK version
 
-**Compatible with [SDK stable ^1.0.0](https://github.com/open-telemetry/opentelemetry-js/tree/stable/v1.0.0) and [SDK experimental ^0.32.0](https://github.com/open-telemetry/opentelemetry-js/tree/experimental/v0.32.0)**
+**Compatible with [SDK stable ^1.0.0](https://github.com/open-telemetry/opentelemetry-js/tree/stable/v1.0.0) and [SDK experimental ^0.34.0](https://github.com/open-telemetry/opentelemetry-js/tree/experimental/v0.34.0)**
```

Comment:
Also here, I think it should be updated to `^1.8.0` as now that we depend on API ^1.3.0, core packages below 1.8.0 are not expected to work correctly (or do they? 🤔)